more/free-programming-cheatsheets.md

```diff
@@ -81,6 +82,15 @@
 * [Cheatsheets for Data Scientists](https://www.datacamp.com/community/data-science-cheatsheets) - Datacamp (PDF)
 
 
+### <a id="data-structures-and-algorithms"></a>Data Structures and Algorithms
```

Comment:
for future reference, the anchor is not needed, it gets generated by the markdown processor.